cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -115,7 +116,7 @@ data StakeAddressCmd
   | StakeRegistrationCert StakeIdentifier OutputFile
   | StakeCredentialDelegationCert
       StakeIdentifier
-      (VerificationKeyOrHashOrFile StakePoolKey)
+      DelegationTarget
```

Comment:
`DelegationTarget` is a wrapper type around `VerificationKeyOrHashOrFile StakePoolKey`.